lib/Cargo.toml

```diff
@@ -116,6 +116,10 @@ trice = "0.3.1"
 ulid = { version = "1.0.0", features = ["serde"] }
 url = "2.4.0"
 
+surrealml = { git = "https://github.com/surrealdb/surrealml.git", branch = "onnx-integration", optional = true}
```

Comment:
This will make it impossible to push `surrealdb` to crates.io. We should publish `surrealml` to crates.io first before we update and merge this PR.